CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ This changelog adheres to [Keep a CHANGELOG](http://keepachangelog.com/).
 ## Unreleased
 
 * [TT-5815] - Change travis distro to "trusty" so builds continue to work
+* Migrate from Travis to Github Action
```

Comment:
Need to keep the format consistent, with other changelog entries, i.e. you need to include the ticket number

Reply:
The ticket was not created when I create the PR, haha. Updated now.